client/post-editor/editor-media-advanced/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External depencencies
+ */
+import React, { PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { toggleEditorMediaAdvanced } from 'state/ui/editor/media/actions';
+import Dialog from 'components/dialog';
+
+function EditorMediaAdvanced( { visible, toggleVisible } ) {
+	return (
+		<Dialog isVisible={ visible } onClose={ toggleVisible }>
+			WIP
+		</Dialog>
+	);
+}
+
+EditorMediaAdvanced.propTypes = {
+	visible: PropTypes.bool,
+	toggleVisible: PropTypes.func
+};
+
+EditorMediaAdvanced.defaultProps = {
+	visible: false,
+	toggleVisible: () => {}
+};
+
+export default connect( ( state ) => {
+	return {
+		visible: state.ui.editor.media.advanced
+	};
+}, ( dispatch ) => {
+	return bindActionCreators( {
+		toggleVisible: toggleEditorMediaAdvanced
+	}, dispatch );
+} )( EditorMediaAdvanced );
```

Comment:
I think so. In fact, just this morning I was thinking how I sometimes feel dirty even saying "ES6" anymore instead of something like ES2015. These kinds of things seem pretty standard to me now  across the board and calling them ES6 or treating them special seems to imply that they are somehow _not JavaScript_. I think there's enough exposure in our codebase that it's either clear to most people or a little opportunity to learn something they didn't realize about the language itself.

Now if only Safari would catch up to supporting all this stuff…